test/fixtures/service_fixtures.js

```diff
@@ -11,7 +11,7 @@ let random = require('../../app/utils/random')
 module.exports = {
   validServiceResponse: (serviceData = {}) => {
     let defaultCustomBranding = {css_url: 'css url', image_url: 'image url'}
-    let defaultMerchantDetais = {
+    let defaultMerchantDetails = {
```

Comment:
Maybe don’t worry about it now but Mark and I try to factor out all uses of `var` and `let` (unless the variable value is modified, then `let` is fine) in files.

It impacts peformace, but this is a test I guess

Reply:
We've done it anyway, added another commit to this PR for this